examples/starburst_trino_NYCTaxi.ipynb

```diff
@@ -0,0 +1,552 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6dfd32ae-583a-4168-88eb-fc98997f0aec",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "joineddata = nycjanduration_new.inner_join(zonelookup, nycjanduration_new.pulocationid.cast(\"str\") == zonelookup.locationid)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a85caeaa-901b-4d88-b8fa-8c9a379c7274",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "joineddata"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dc9b9427-3ebd-44a0-8f32-f98c9461044f",
+   "metadata": {},
+   "source": [
+    "Now we can do more cool things in ibis with group bys and aggregate by with zones and boroughs!\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "027ddcca-c387-427d-8c64-5a2bd94945df",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "groupbyboroughtrips = (\n",
+    "    joineddata\n",
+    "    .group_by(\"zone\")\n",
+    "    .aggregate(\n",
+    "        trips=joineddata.vendorid.count(),\n",
+    "        totalrev=joineddata.fare_amount.sum(),\n",
+    "        totalpassengers=joineddata.passenger_count.sum(),\n",
+    "        averageride=joineddata.rideminutes.mean().round(2)\n",
+    "        \n",
+    "        )\n",
+    "    .order_by(ibis.desc(\"totalrev\"))\n",
+    "    .limit(10)\n",
+    ")\n",
+    "groupbyboroughtrips\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7cef329c-0cbb-4eed-89fd-3596eb195393",
+   "metadata": {},
+   "source": [
+    "If you want to see what sql ibis generates, you can use the ibis.to_sql() method."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "200492d2-3766-49bd-8a94-bc3b90b23044",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ibis.to_sql(groupbyboroughtrips)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "46d8bc4d-4b9d-4e2d-aa5b-c059ec9a6ad2",
+   "metadata": {},
+   "source": [
+    "Airport rides give the most revenue to taxi companies, that makes a lot of sense. What about by week? Any change in behaviour over the month? \n",
+    "P.S This ibis code below at the time of writing uses a pre-release function \"bucket\". This is included with future version of IBIS. When bucket is released, you can uncomment the lines below for a temporal grouping. If you recieve an error \n",
+    "the function does not exist, you can uncomment the line below to try to upgrade to a pre-release version of ibis:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1f8f9893-a369-4337-8c86-38045e75a461",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#python -m pip install -U 'ibis-framework[trino]' --pre"
+   ]
+  },
```

Comment:
Sorry for the delay in getting 7.1 released -- I think we should hold off on publishing this until we don't require a pre-release feature.